EasyFinance.Server/Controllers/ExpensesController.cs

```diff
@@ -22,71 +23,49 @@ public ExpensesController(IExpenseService expenseService, UserManager<User> user
             this.userManager = userManager;
         }
 
-
         [HttpGet]
-        public async Task<IActionResult> Get(Guid categoryId, DateTime from, DateTime to)
+        public async Task<IActionResult> Get(Guid categoryId, DateTime from, DateTime to, Paging paging)
         {
-            var expenses = await this.expenseService.GetAsync(categoryId, from, to);
-            return Ok(expenses.ToDTO());
+            var expenses = await expenseService.GetAsync(categoryId: categoryId, from: from, to: to, paging: paging);
+            return Ok(expenses);
```

Comment:
```suggestion
            return ValidateResponse(result, HttpStatusCode.OK);
```
After add a base controller you can use it to help you.